### PR TITLE
DO-94: Removed Modal from Picture Viewer.

### PR DESCRIPTION
**Pull request by mohitsaladiya**

### CODE REVIEW

File: src/components/common/PictureViewer/DeleteImageButton/DeleteImageButton.jsx [hunk 1]
```diff
@@ -134,16 +135,10 @@ class DeleteImageButton extends React.Component {
...
-        style={{
-          backgroundColor: 'transparent',
-          borderColor: 'transparent',
-          color: 'inherit',
-        }}
-        onClick={this.handleImageDelete}
-      >
-        <Icon style={{ fontSize: '30px' }} icon={faTrash} />
-        <div>Delete</div>
+      <button className="remove-image-button" onClick={this.handleImageDelete}>
```
Comment (mohitsaladiya) [inline src/components/common/PictureViewer/DeleteImageButton/DeleteImageButton.jsx:138]: Use Button component plus use proper classNames as per convention.

File: src/components/common/PictureViewer/PictureViewer.jsx [hunk 2]
```diff
@@ -118,10 +118,10 @@ class PictureViewer extends React.Component {
     // Renders custom controls only if you have imageUrl
     if (this.getImageUrlAtIndex()) {
       return (
-        <>
+        <div className="header-button-container">
```
Comment (mohitsaladiya) [inline src/components/common/PictureViewer/PictureViewer.jsx:121]:
```suggestion
        <div className="picture-viewer__header-buttons-container">

```
```

File: src/components/common/PictureViewer/ReplaceImageButton/ReplaceImageButton.jsx [hunk 3]
```diff
@@ -27,9 +28,8 @@ const ReplaceImageButton = (props) => {
         onUploadSuccess={onUploadSuccess}
         selectedImageId={selectedImageId}
       >
-        <div>
-          <Icon style={{ fontSize: '30px' }} icon={faPencilAlt} />
-          <div>Replace</div>
+        <div className="action-icon-container">
```
Comment (mohitsaladiya) [inline src/components/common/PictureViewer/ReplaceImageButton/ReplaceImageButton.jsx:31]:
```suggestion
        <div className="replace-image-button__container">

```

File: src/components/member/MemberPictureViewerAndUploader/MemberPictureUploader/MemberPictureUploader.jsx [hunk 4]
```diff
@@ -41,9 +42,9 @@ const MemberPictureUploader = (props) => {
         onUploadError={onUploadError}
         selectedImageId={selectedImageId}
       >
-        <div>
-          <Icon style={{ margin: '10px', fontSize: '60px' }} icon={faUpload} />
-          <div>Click to Upload your image now!</div>
+        <div className="upload-msg">
```
Comment (mohitsaladiya) [inline src/components/member/MemberPictureViewerAndUploader/MemberPictureUploader/MemberPictureUploader.jsx:45]:
```suggestion
        <div className="member-picture-uploader__upload-msg">
```

File: src/components/common/PictureViewer/DeleteImageButton/DeleteImageButton.jsx (hunk 1 above)
Comment (amitjain24524) [reply src/components/common/PictureViewer/DeleteImageButton/DeleteImageButton.jsx]: Done. Class name updated.

File: src/components/common/PictureViewer/PictureViewer.jsx (hunk 2 above)
Comment (amitjain24524) [reply src/components/common/PictureViewer/PictureViewer.jsx]: Done

File: src/components/common/PictureViewer/ReplaceImageButton/ReplaceImageButton.jsx (hunk 3 above)
Comment (amitjain24524) [reply src/components/common/PictureViewer/ReplaceImageButton/ReplaceImageButton.jsx]: Done

File: src/components/member/MemberPictureViewerAndUploader/MemberPictureUploader/MemberPictureUploader.jsx (hunk 4 above)
Comment (amitjain24524) [reply src/components/member/MemberPictureViewerAndUploader/MemberPictureUploader/MemberPictureUploader.jsx]: Done

**Pull request merged**